sql/mob_droplist.sql

```diff
@@ -15956,7 +15956,7 @@ INSERT INTO `mob_droplist` VALUES (3054,0,0,1000,2848,222); -- (Trembler Tabitha
 INSERT INTO `mob_droplist` VALUES (3055,0,0,1000,16018,333); -- (Naa Zeku the Unwaiting) Ataraxy Earring
 INSERT INTO `mob_droplist` VALUES (3056,0,0,1000,17970,100); -- (Centipedal Centruroides) Lyft Tabar
 INSERT INTO `mob_droplist` VALUES (3057,0,0,1000,16297,62); -- (Okyupete) Shepherd's Chain
-INSERT INTO `mob_droplist` VALUES (3058,0,0,1000,2841,444); -- (Be'Hya Hundredwall) Quadav Silver Ingot
+INSERT INTO `mob_droplist` VALUES (3058,0,0,1000,2841,266); -- (Be'Hya Hundredwall) Quadav Silver Ingot
```

Comment:
http://ffxidb.com/zones/143/behya-hundredwall

Seems to fall into very common (240)